app/helpers/cms_helper.rb

```diff
@@ -181,6 +181,7 @@ def get_resource(fragment_text, fragment_link, button_text, button_class)
   # Also sanitises it in the case of a download link
   def linkify(fragment_link)
     if fragment_link.tag == 'file'
+      return '' if !fragment_link.attachments || fragment_link.attachments.first.blank?
```

Comment:
It would be better to be consistent with what we return in case of missing links or attachments. So I would suggest `nil` both here and below.

Reply:
OK, I have made the changes